src/EventStore.ClusterNode/ClusterVNodeHostedService.cs

```diff
@@ -153,13 +132,25 @@ internal class ClusterVNodeHostedService : EventStoreHostedService<ClusterNodeOp
 					"\n DEPRECATION WARNING: AtomPub over HTTP Interface has been deprecated as of version 20.6.0. It is recommended to use gRPC instead.\n");
 			}
 
+			if (opts.DisableInternalTcpTls) {
+				Log.Warning(
+					$"\n DEPRECATION WARNING: The '{nameof(Options.DisableInternalTcpTls)}' option has been deprecated as of version 20.6.1. "
```

Comment:
We should look at displaying these warnings a little better in the future...
Instead of
```
[23312, 1,08:18:55.988,WRN]
 DEPRECATION WARNING: AtomPub over HTTP Interface has been deprecated as of version 20.6.0. It is recommended to use gRPC instead.

[23312, 1,08:18:55.988,WRN]
 DEPRECATION WARNING: The Legacy TCP Client Interface has been deprecated as of version 20.6.0. The External TCP Interface can be re-enabled with the 'EnableExternalTCP' option. It is recommended to use gRPC instead.
```

```
[23312, 1,08:18:55.988,WRN]
 DEPRECATION WARNING: 
  - AtomPub over HTTP Interface has been deprecated as of version 20.6.0. It is recommended to use gRPC instead.
  - The Legacy TCP Client Interface has been deprecated as of version 20.6.0. The External TCP Interface can be re-enabled with the 'EnableExternalTCP' option. It is recommended to use gRPC instead.
```